engines/director/lingo/lingo-funcs.cpp

```diff
@@ -355,7 +355,7 @@ void Lingo::func_cursor(int c) {
 }
 
 void Lingo::func_beep(int repeats) {
-	for (int r = 0; r <= repeats; r++)
+	for (int r = 1; r <= repeats; r++)
```

Comment:
I would rather change it back to `0` but change the `<=` condition to `<`

Reply:
I would prefer to keep it at `1` and `<=` , so the delay can be `if (r < repeats) delay()` instead of `if (r  < repeats - 1)`